.gitlab/pipeline/test.yml

```diff
@@ -92,7 +92,24 @@ test-linux-stable-additional-tests:
         --locked \
         --release \
         --features runtime-benchmarks,try-runtime
-  allow_failure: true
+    # https://github.com/paritytech/ci_cd/issues/864
+    - time cargo nextest run --features runtime-benchmarks benchmark --locked --cargo-profile testnet
```

Comment:
Why is this run both here and below in the new job?

Reply:
Nice catch, forgot to remove the duplicate, thanks